Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -328,6 +332,8 @@ - (void)encodeWithCoder:(NSCoder *)coder
 #else
     [coder encodeInteger:self.userInterfaceDirectionPolicy forKey:@"userInterfaceDirectionPolicy"];
 #endif
+    [coder encodeBool:self._scrollToTextFragmentIndicatorEnabled forKey:@"scrollToTextFragmentIndicatorEnabled"];
+    [coder encodeBool:self._scrollToTextFragmentMarkingEnabled forKey:@"scrollToTextFragmentMarkingEnabled"];
```

Comment:
Not sure if this is needed.

Reply:
There's no reason to exclude it. We want encode/decode of `WKWebViewConfiguration` to produce equivalent objects.